Rn 1.5.0 (#398)
percona/percona-server-mongodb-operator: source/RN/Kubernetes-Operator-for-PSMONGODB-RN1.5.0.rst

.. _K8SPSMDB-1.5.0:

================================================================================
*Percona Kubernetes Operator for Percona Server for MongoDB* 1.5.0
================================================================================

> :Date: September 10, 2020

Reviewer: The release date was changed and is currently set to 7th so please change it.

.. _K8SPSMDB-1.5.0:

================================================================================
*Percona Kubernetes Operator for Percona Server for MongoDB* 1.5.0
================================================================================

:Date: September 10, 2020
:Installation: `Installing Percona Kubernetes Operator for Percona Server for MongoDB <https://www.percona.com/doc/kubernetes-operator-for-psmongodb/index.html#installation>`_

New Features
================================================================================

* :jirabug:`K8SPSMDB-233`: Automatic management of system users for MongoDB on password rotation via Secret
* :jirabug:`K8SPSMDB-226`: Official Helm chart for the Operator
* :jirabug:`K8SPSMDB-199`: Support multiple PSMDB minor versions by the Operator
* :jirabug:`K8SPSMDB-198`: Fully Automate Minor Version Updates (Smart Update)

Improvements
================================================================================

> * :jirabug:`K8SPSMDB-192`: The ability to set the mongod cursorTimeoutMillis parameter in YAML (Thanks to user xprt64 for reporting this issue)

Reviewer 2: ```suggestion
* :jirabug:`K8SPSMDB-192`: The ability to set the mongod cursorTimeoutMillis parameter in YAML (Thanks to user xprt64 for the contribution)
```

> * :jirabug:`K8SPSMDB-197`: Adding additional certificate SANS useful for reverse DNS lookups

Reviewer 2: ```suggestion
* :jirabug:`K8SPSMDB-197`: Adding additional certificate SANS useful for reverse DNS lookups (Thanks to user phin1x for the contribution)
```

> * :jirabug:`K8SPSMDB-190`: Direct API quering with "curl" instead of using "kubectl" tool in scheduled backup jobs

Reviewer 2: ```suggestion
* :jirabug:`K8SPSMDB-190`: Direct API quering with "curl" instead of using "kubectl" tool in scheduled backup jobs (Thanks to user phin1x for the contribution)
```

* :jirabug:`K8SPSMDB-133`: A special PSMDB debug image which avoids restarting on fail and contains additional tools useful for debugging
* :jirabug:`K8SPSMDB-146`: PSMDB backup operator needs to have tar binary added (Thanks to user sdotel for reporting this issue)

> * :jirabug:`CLOUD-556`: Kubernetes 1.17 added to the list of supported platforms

Reviewer 2: I think K8SPSMDB-234 also should be here

Bugs Fixed
================================================================================

* :jirabug:`K8SPSMDB-213`: Installation instruction not reflecting recent changes in git tags (Thanks to user geraintj for reporting this issue)
* :jirabug:`K8SPSMDB-246`: backup failing if backup user's password rotated

> * :jirabug:`K8SPSMDB-245`: users password rotation - error on updating PMM server user in PSMDB

Reviewer 2: both issues were a consequence of K8SPSMDB-233, so should be excluded

.. _K8SPSMDB-1.5.0:

================================================================================
*Percona Kubernetes Operator for Percona Server for MongoDB* 1.5.0
================================================================================

:Date: August 27, 2020
:Installation: `Installing Percona Kubernetes Operator for Percona Server for MongoDB <https://www.percona.com/doc/kubernetes-operator-for-psmongodb/index.html#installation>`_

New Features
================================================================================

* :jirabug:`K8SPSMDB-199`: Support multiple PSMDB minor versions by the Operator
* :jirabug:`K8SPSMDB-226`: Official Helm chart for the Operator
* :jirabug:`K8SPSMDB-198`: Fully Automate Minor Version Updates (Smart Update)

> 

Reviewer 2: also we need to add https://jira.percona.com/browse/K8SPSMDB-233
